.github/workflows/content-release-prod.yml

```diff
@@ -0,0 +1,21 @@
+name: "Content release: Prod"
+on:
+  # Can be manually triggered
+  workflow_dispatch:
```

Comment:
If we go to production, we probably probably don't want to be able to trigger a build manually, maybe. Wanted to call that concern out there.

Reply:
We have times where we need to fire Content Build manually if the normal triggers are failing. I think it's ok.

However, that brings up a point I missed: these workflows need concurrency management. I'll add that in.